src/client/constants/paths.js

```diff
@@ -6,7 +6,7 @@ const PATHS = {
   createTripOld: (userId = ':userId') => `/user/${userId}/create-trip`,
   swipe: '/swipe',
   swipeOld: (userId = ':userId', tripId = ':tripId') => `/user/${userId}/trip/${tripId}/swipe`,
-  trips: (userId = ':userId') => `/user/${userId}/trips`,
+  trips: '/trips',
```

Comment:
so now the user id we have to get from the token? so we just remove from the url?

Reply:
yep!